scripts/generate_matrix.py

```diff
@@ -0,0 +1,227 @@
+# !/usr/bin/env python3
```

Comment:
Can this be removed?

Reply:
Yes, we don't really need it in this case.